src/screens/HomeScreen.jsx

```diff
@@ -13,27 +13,37 @@ import { CatCard } from '../CatCard/CatCard';
 import { cats } from '../data/cats';
 import { styles } from '../styles';
 
-const renderSeparator = () => (<View style={styles.separator} />);
-
-const renderCatItem = ({ item }, navigation) => (
-  <TouchableOpacity onPress={() => (navigation.navigate('Details', { catId: item.id }))}>
-    <CatCard cat={item} />
-  </TouchableOpacity>
-);
-
-export const HomeScreen = ({ navigation }) => (
-  <SafeAreaView>
-    <FlatList
-      style={styles.container}
-      data={cats}
-      renderItem={(item) => renderCatItem(item, navigation)}
-      ItemSeparatorComponent={renderSeparator}
-    />
-  </SafeAreaView>
-);
-
-renderCatItem.propTypes = {
-  item: PropTypes.objectOf(PropTypes.string).isRequired,
+export const HomeScreen = ({ navigation }) => {
+  const openDetailedCatCard = (item) => (navigation.navigate('Details', { catId: item.id }));
+
+  const renderCatItem = ({ item }) => (
+    <TouchableOpacity onPress={() => openDetailedCatCard(item)}>
+      <CatCard cat={item} />
+    </TouchableOpacity>
+  );
+
+  renderCatItem.propTypes = {
+    item: PropTypes.objectOf(PropTypes.string).isRequired,
+    navigation: PropTypes.shape({
+      navigate: PropTypes.func.isRequired,
+    }).isRequired,
+  };
+
+  const renderSeparator = () => (<View style={styles.separator} />);
```

Comment:
As this code doesn't refer to values inside component then it can be moved outside component